src/SSHDebugPS/RawShell.cs

```diff
@@ -0,0 +1,295 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using liblinux;
+using liblinux.Shell;
+using Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier;
+
+namespace Microsoft.SSHDebugPS
+{
+    public interface IRawShell : IDisposable
+    {
+        event EventHandler<string> OutputReceived;
+        event EventHandler<int> Closed;
+        event EventHandler ErrorOccured;
+
+        void WriteCommandStart(string startCommand);
```

Comment:
>void WriteCommandStart(string startCommand); [](start = 8, length = 44)

It looks like all implementations of this just call `WriteLine` - remove from interface?